### ISSUE TITLE
XPaths for geospatial_vertical_units, geospatial_vertical_resolution, geospatial_vertical_positive

### ISSUE THREAD

**Issue by n-a-t-e** [cioos-siooc/metadata-xml#12]
It's hard to tell where to stick these variables. Here are some example values for them:

`geospatial_vertical_units`: 'EPSG:5829'
`geospatial_vertical_resolution`:  '25 meters'
`geospatial_vertical_positive`: 'up'

VerticalCRS seems like the spot, as attributes or child elements: `/mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:extent/gex:EX_Extent/gex:verticalElement/gex:EX_VerticalExtent/gex:verticalCRS`

For `geospatial_vertical_units`, @fostermh suggested:
```
<gex:verticalCRS xlink:href="http://www.epsg-registry.org/export.htm?gml=urn:ogc:def:crs:EPSG::5862" xlink:title="LLWLT depth in meters" xlink:actuate="onRequest"/>
```

There are some good examples on this page of child elements would could also use:
https://geo-ide.noaa.gov/wiki/index.php?title=GML_Guidance_for_ISO_Metadata#gml:VerticalCRS

**Comment by fostermh** [comment]
also the crs we pick will may dictate the vertical positive direction, for
example, EPSG:5829 is "Height relative to instantaneous water level
uncorrected for tide. Not specific to any location or epoch." so "up"
whereas EPSG:5862 is "Lower Low Water Large Tide depth" so "down"


Matthew Foster
Hakai Institute, Calvert Island, BC, Canada / www.hakai.org
Tula Foundation, Quadra Island, BC, Canada / www.tula.org

**Comment by jeffcullis** [comment]
Yes like Matt said it looks like the CRS we pick will dictate this. From [here](https://gis.stackexchange.com/questions/6484/is-there-a-standard-vertical-datum-for-meters-below-the-wgs84-ellipsoid-or-belo) it looks like [EPSG:5829](http://epsg.io/5829) is "up" for water height (unforrected for tide) while [EPSG:5831](http://epsg.io/5831) is "down" for water depth.
Not sure about the tide corrections but presumably we're not often correcting for tide?

**Comment by n-a-t-e** [comment]
This was fixed by #64
